src/components/auth/ResetPasswordForm.tsx

```diff
@@ -48,16 +48,21 @@ function ResetPasswordForm(props: Props) {
         icon="lock"
         iconPosition="left"
         error={
-          validator.isEmpty(passwordRepeat) || !validator.equals(passwordRepeat, password)
+          !passwordIsValid || !validator.equals(passwordRepeat, password)
         }
         label={t('pages.resetPassword.repeatPassword')}
         onChange={(e: ChangeEvent<HTMLInputElement>) => changePasswordRepeat(e.target.value)}
       />
+      <PasswordStrength
+        password={password}
+        setPasswordIsValid={setPasswordIsValid}
+      />
       <Button
         fluid
         primary
         size="large"
         type="submit"
+        disabled={validator.isEmpty(passwordRepeat) || !validator.equals(passwordRepeat, password) || !validator.isStrongPassword(password)}
```

Comment:
Instead of `validator.isStrongPassword`, why not use `passwordIsValid`? If `isStrongPassword` implements different constraints, the user has no clue what is wrong.